deploy.cfg

```diff
@@ -16,9 +16,6 @@ mongo-database = handle_db
 mongo-user =
 mongo-password =
 
-# start local mongod service
```

Comment:
I assume this isn't needed any more... it was causing an extra mongo version to start

Reply:
👍 I cannot remember why we have start_local. Maybe for some local testing purposes.